src/bin-utils/index.js

```diff
@@ -4,13 +4,15 @@ import {remove, includes, isPlainObject, isEmpty} from 'lodash'
 import shellEscape from 'shell-escape'
 import colors from 'colors/safe'
 import {safeLoad} from 'js-yaml'
+import tree from 'ascii-tree'
 
 import getLogger from '../get-logger'
 import {resolveScriptObjectToScript} from '../resolve-script-object-to-string'
 import initialize from './initialize'
 import {default as autocomplete, install as installAutocomplete} from './autocomplete'
 
 const log = getLogger()
+const indentCharacter = '#'
```

Comment:
I'm confused by this. I don't see the character `#` in any of the output. Is this something special to `ascii-tree` or something?